### ISSUE TITLE
Customizable Validation errors is planning?

### ISSUE THREAD

**Issue by roquie** [vapor/vapor#2513]
https://github.com/vapor/validation/issues/22

**Comment by 0xTim** [comment]
This has come up several times so definitely popular. However it will require significant rework to the validation engine

**Comment by roquie** [comment]
I think is a require to do in near future... Because internalization is important feature.

**Comment by 0xTim** [comment]
Yes definitely worth doing when we come. If you want to try, PRs are gratefully accepted!
